### PR TITLE
KYLIN-3552 Data Source SDK to ingest data from different JDBC sources

### PR DESCRIPTION
**Pull request by edouardzyc**

### CODE REVIEW

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

File: examples/test_case_data/localmeta/data/DEFAULT.TEST_ACCOUNT.csv [hunk 1]
```diff
@@ -0,0 +1,2000 @@
+10000000,3,2,GB,ACCO603
```
Comment (lidongsjtu) [inline examples/test_case_data/localmeta/data/DEFAULT.TEST_ACCOUNT.csv]: remove these files

File: server-base/pom.xml [hunk 2]
```diff
@@ -72,7 +72,21 @@
...
+        </dependency>
+        <dependency>
+            <groupId>org.apache.kylin</groupId>
+            <artifactId>kylin-datasource-sdk</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.kylin</groupId>
+            <artifactId>kylin-datasource-sdk</artifactId>
+            <version>${project.version}</version>
```
Comment (lidongsjtu) [inline server-base/pom.xml]: remove version

**Comment by shaofengshi** [comment]
As there is conflict, I manually merged this pr with patch; Thanks yicheng and dong!

**Pull request closed without merging**